src/AppHeader.js

```diff
@@ -0,0 +1,24 @@
+function AppHeader({ inputChange, addItem, inputValue }) {
+    return (
+        <div className='header'>
+            <h2 className='header__title'>To Do List</h2>
+            <div className='header__input'>
+                <input
+                    className='input'
+                    type='text'
+                    id='input'
+                    placeholder='New Task...'
+                    onChange={inputChange}
+                    value={inputValue}
+                />
+                <span
+                    className='add-button'
+                    id='addButton'
+                    onClick={addItem}
```

Comment:
here you add and onClick, however there is a popular pattern on the web submit value with pressing Enter which you not supporting here.

I believe the easiest way for you to achieve it would be wrapping this button and input above in `form`, add to `form` onSubmit handler where you would make `e.preventDefault()` which would help you to avoid page refresh and then you call `addItem`. Also you need to add a button type="submit"

Reply:
I used here form and it works now fine.